### PR TITLE
EBS test nav

### PR DESCRIPTION
**Pull request by b1n-ch1kn**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimisation

## [required] Description

Previous testing of control in a local reference frame for EBS test has proven limited. Hoping for more stable control with a global reference frame given improvements to SLAM and cone mapping.

## [required] Documentation

Not a feature - param changes documented in config and launch files

### [optional] Usability concerns or breaking changes?

None

### [optional] Screenshots

### CODE REVIEW

File: src/common/vehicle_urdf/urdf/sensors.urdf.xacro
Comment (leungw16) [inline src/common/vehicle_urdf/urdf/sensors.urdf.xacro:1]: Maybe follow the existing vehicle urdf for consistency?
https://github.com/QUT-Motorsport/QUTMS_Driverless/blob/b9ab06eb14bb6c986285fe97e384ec21b1fc2285/src/common/vehicle_urdf/urdf/vehicle.urdf.xacro#L19

Comment (b1n-ch1kn) [reply src/common/vehicle_urdf/urdf/sensors.urdf.xacro:1]: I was considering that yeah, I think the 16 is actually smaller than the 32 so this cylinder should be the proper dimensions for our lidar (plus its just a cylinder)

**Review by leungw16 — APPROVED** [review]
LGTM

**Comment by AlexCB192** [comment]
Testing plan looks good.

**Review by AlexCB192 — APPROVED** [review]

**Pull request merged**